src/database/models/team_item.rs

```diff
@@ -100,11 +108,230 @@ impl TeamMember {
                 user_id: UserId(m.user_id),
                 name: m.member_name,
                 role: m.role,
+                permissions: Permissions::from_bits_truncate(m.permissions as u64),
```

Comment:
This should not truncate, since it's loading the data; using truncate when checking permissions is fine, but using it here could cause data loss.